mkdocs.yml

```diff
@@ -217,9 +247,15 @@ markdown_extensions:
   - pymdownx.mark
   - pymdownx.tilde
   - attr_list
+  - md_in_html
   - tables
   # - mdx_truly_sane_lists
 extra:
+  version:
+    provider: mike
+  # - make edgexversion the latest version for repository tree references
+  edgexversion: 'v3.0.0'
+  edgexversionmain: 'main'
```

Comment:
Do we still need the template_macros file anymore then? https://github.com/edgexfoundry/edgex-docs/blob/main/template_macros.yaml. Can we combine everything in this extras section?

Reply:
@ernestojeda - I believe we can get rid of them as it doesn't appear the mkdocs is able to use these macros (at least not with mike in place).  This is probably something for research.  And we may want to organize the macros in mkdocs a bit better once we understand what we need.

With this PR, we can start to refine and improve.

Reply:
Yes, remove it since it is no longer needed.

Reply:
returned version-select.js and renamed version to edgexversion in the template macros per @lenny-intel suggestion.